client-choo/components/public-key-card.js

```diff
@@ -0,0 +1,99 @@
+var html = require('choo/html')
+var Component = require('choo/component')
+const validUrl = require('valid-url')
+const createIcon = require('blockies-npm')
+
+const avatar = require('../../client/utils/avatar')
+
+const { OBJECT, STRING, UNDEFINED,
+        ARRAY, INTEGER, BOOL } = require('../../src/utils')
+
+const OP_UNFOLLOW = 'Unfollow'
+const OP_FOLLOW = 'Follow'
+
+module.exports = class PublicKeyCard extends Component {
+  constructor (id, state, emit) {
+    super(id)
+    this.state = state
+    this.emit = emit
+  }
+
+  update (newState) {
+    this.state = newState
+    return true
+  }
+
+  close () {
+    this.emit('closePublicKeyCard')
+  }
+
+  follow (event) {
+    const that = this
+    if (event.target.dataset.operation) {
+      if (event.target.dataset.operation === OP_UNFOLLOW) {
+        this.unfollow()
+        return
+      }
+    }
+    this.emit('followPeer')
+  }
+
+  unfollow () {
+    this.emit('unfollowPeer')
+  }
+
+  createElement (state) {
+    const { IpfsID, publicKeyCard: { profile, validityDocs, invalidDocs } } = state
+    const icon = avatar(profile.peerId)
+    let followBtn
+    if (profile.peerId === IpfsID.identity.profile.peerId) {
+      followBtn = { disabled: 'disabled', label: '(You)' }
+    } else if (profile.following) {
+      followBtn = { disabled: '', label: OP_UNFOLLOW }
+    } else {
+      followBtn = { disabled: '', label: OP_FOLLOW }
+    }
+    // had to do this because of a really weird bug in the bundler, where it was lower-casing the `followBtn` var to `followbtn`
```

Comment:
was that a Browserify thing? was there an issue in their project repo?

Reply:
I didn't dig into it. I just looked at the bundle.js file and it had a lower-cased version of the variable. I searched the codebase and it was always camel-cased in the source. Not sure why that was happening.